app/Http/Controllers/PayloadController.php

```diff
@@ -73,7 +73,10 @@ public function store(PayloadCreateRequest $request, $webhookId)
             }
 
             DB::commit();
-            $request->session()->flash('messageSuccess', 'This payload successfully created');
+            $request->session()->flash('messageSuccess', [
+                'status' => 'Create success',
+                'message' => 'This payload successfully created'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadController.php</br>**Line**: 78</br>Each line in an array declaration must end in a comma